apps/web/app/(auth)/forgot-password/loading.tsx

```
@@ -0,0 +1,23 @@
+import { Skeleton } from '@taboot/ui/components/skeleton';
+
+export default function ForgotPasswordLoading() {
+  return (
+    <div className="from-background to-muted/20 flex min-h-screen items-center justify-center bg-linear-to-b px-4" aria-busy="true" aria-label="Loading forgot password">
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)